packages/block-library/src/details/block.json

```diff
@@ -0,0 +1,13 @@
+{
+	"name": "core/details",
+	"category": "layout",
+	"attributes": {
+		"initialOpen": {
+			"type": "boolean",
+			"default": false
+		},
+		"content": {
+			"type": "string"
+		}
```

Comment:
This isn't really the "content" of the block, but rather just the content of the summary. I'd rename this attribute to "summaryContent".